prow/cmd/deck/main.go

```diff
@@ -488,3 +574,26 @@ func handleBranding(ca configAgent) http.HandlerFunc {
 		}
 	}
 }
+
+func isValidatedGitOAuthConfig(githubOAuthConfig *config.GithubOAuthConfig) bool {
+	return githubOAuthConfig.ClientID != "" && githubOAuthConfig.ClientSecret != "" &&
+		githubOAuthConfig.RedirectURL != "" &&
+		githubOAuthConfig.FinalRedirectURL != ""
+}
+
+func getUserDashboard(path string) (userdashboard.UserData, error) {
```

Comment:
`getUserData` might be a more appropriate name.

Reply:
done